credentials/local/local.go

```diff
@@ -0,0 +1,105 @@
+/*
+ *
+ * Copyright 2018 gRPC authors.
```

Comment:
2020

Reply:
Hmm ... Is it possible for our vet to handle this? I'm seeing this quite a bit :)

Reply:
I don't see why not.  We should be able to get the current year with `date +%Y`.

Reply:
The hard part is to find if the modified lines.
vet.sh matches all files, if we do `date +%Y`, it's going to complain about untouched files/unmodified lines.

Reply:
We would only want to do it for new files vs. the master branch.  I think we could pull that off.

Reply:
What if I move a file? Copyright will be kept, but it shows as a new file (or not?)

Reply:
:thinking:  If it's not possibly to distinguish those, then we could annotate the commit message to make it ignore certain files.  It might not be worth it at that point.